vanilla-js/scripts/run-example.js

```diff
@@ -27,9 +27,10 @@ process.env.DESCRIPTION = example.description;
 process.env.FILENAME = example.filename;
 
 // Run the example
+// removed internal npm call to run-example and instead added the commands directly here aas env variables are not getting passed"
 try {
-  execSync("npm run --silent run-example", { stdio: "inherit", timeout: 20000 }); // 20-second timeout
+  execSync("echo \"=== $TITLE ===\" && node ./scripts/typewriter.js \"$DESCRIPTION\" && bat --language=js --theme=\"OneHalfDark\" $FILENAME && node $FILENAME", { stdio: "inherit", timeout: 20000 }); // 20-second timeout
```

Comment:
We can also ditch the typewriter thing if it's too error-prone.

Also if the example is too long, the user has to arrow down and hit `q` to get out of it. Maybe we should rethink how the examples run? We can pair next SDK Sync

Reply:
Agreed. When the examples are long it takes time to wrap up the text. For now I will keep as is and we can discuss it over our next sync.